site/js/expander.js

```diff
@@ -1,25 +1,14 @@
-// 	event handlers on each button
+// 	'See All' button event handlers on Documentation page
 //	on click hide elem, show all
 
-const buttons = document.querySelectorAll('button[id^="expandButton-"]');
-let i;
-
-for (i = 0; i < buttons.length; i++) {
-  buttons[i].addEventListener('click', function () {
-
-    const classId = this.id.replace('expandButton-', '');
-    const expanded = document.getElementById('expanded-' + classId);
-    const collapsed = document.getElementById('collapsed-' + classId);
-
-    if (expanded.className === 'none') {
-      expanded.className = 'inline';
-      collapsed.className = 'none';
-      this.textContent = 'See fewer';
-    } else if (expanded.className === 'inline') {
-      expanded.className = 'none';
-      collapsed.className = 'inline';
-      this.textContent = 'See all';
+document.getElementById('Documentation').addEventListener('click', (e) => {
+  if (e.target && e.target.className.includes('expand-button')) {
+    if (e.target.parentNode.children[0].className === 'none') {
+      e.target.parentNode.children[0].className = 'inline';
```

Comment:
If you use the `classList` API here, you won't be overriding other classes that might exist on the element. See https://developer.mozilla.org/en-US/docs/Web/API/Element/classList